### PR TITLE
feat: Added limiters for CG fields

### PR DESCRIPTION
**Pull request by bs-amitmahajan29**
https://browserstack.atlassian.net/browse/DATA-7483
https://browserstack.atlassian.net/wiki/spaces/ENG/pages/4039344275/CG+log+limits+over+text+fields

### CODE REVIEW

File: lib/chitragupta/field_limits.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  DYNAMIC_DATA: 5000,
+  HEADERS: 1000,
+  PARAMS: 10000,
```
Comment (bs-amitmahajan29) [inline lib/chitragupta/field_limits.js:4]:
```suggestion
  DYNAMIC_DATA: 10,
  HEADERS: 20,
  PARAMS: 25,
```

File: lib/chitragupta/field_limits.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  DYNAMIC_DATA: 10,
+  HEADERS: 20,
+  PARAMS: 25,
```
Comment (bs-amitmahajan29) [inline lib/chitragupta/field_limits.js]:
```suggestion
  DYNAMIC_DATA: 10,
  HEADERS: 5,
  PARAMS: 8,
```

**Review by rohit3463 — APPROVED** [review]

**Comment by rohit3463** [comment]
RUN_TESTS

**Review by rohit3463 — DISMISSED** [review]

File: lib/chitragupta/field_limits.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  DYNAMIC_DATA: 4000,
```
Comment (bs-amitmahajan29) [inline lib/chitragupta/field_limits.js]:
```suggestion
  DYNAMIC_DATA: 5000,
```

**Review by sunny141193 — APPROVED** [review]

**Pull request merged**